### PR TITLE
Updated assimp to commit 1d565b0 with iFire

### PR DESCRIPTION
**Pull request by RevoluPowered**
Signed-off-by: RevoluPowered <gordon@gordonite.tech>
Signed-off-by: K. S. Ernest (iFIre) Lee <ernest.lee@chibifire.com>

### CODE REVIEW

File: thirdparty/assimp/code/HMP/HMPLoader.h [hunk 1]
```diff
@@ -0,0 +1,153 @@
+/*
```
Comment (fire) [inline thirdparty/assimp/code/HMP/HMPLoader.h]: Remove HMP loader.

File: thirdparty/assimp/code/Irr/IRRLoader.h [hunk 2]
```diff
@@ -0,0 +1,296 @@
+/*
```
Comment (fire) [inline thirdparty/assimp/code/Irr/IRRLoader.h]: Remove IRR loader.

File: thirdparty/assimp/code/Assxml/AssxmlExporter.cpp [hunk 3]
```diff
@@ -0,0 +1,655 @@
+/*
```
Comment (fire) [inline thirdparty/assimp/code/Assxml/AssxmlExporter.cpp]: Remove ASSXML exporter

File: thirdparty/assimp/include/assimp/irrXMLWrapper.h [hunk 4]
```diff
@@ -44,7 +44,11 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #define INCLUDED_AI_IRRXML_WRAPPER
```
Comment (fire) [inline thirdparty/assimp/include/assimp/irrXMLWrapper.h]: Probably not needed.

**Comment by fire** [comment]
```In file included from thirdparty/assimp/include/assimp/defs.h:53,
                 from thirdparty/assimp/include/assimp/types.h:58,
                 from thirdparty/assimp/include/assimp/Logger.hpp:49,
                 from thirdparty/assimp/include/assimp/DefaultLogger.hpp:48,
                 from modules/assimp/editor_scene_importer_assimp.h:47,
                 from modules/assimp/register_types.cpp:34:
thirdparty/assimp/assimp/config.h:651:1: error: "/*" within comment [-Werror=comment]
  651 | /** @brief  Set wether the importer shall not remove empty bones.
      |  
cc1plus: all warnings being treated as errors```

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**